src/MagnumPlugins/UfbxImporter/Test/UfbxImporterTest.cpp

```diff
@@ -218,6 +224,142 @@ void UfbxImporterTest::mesh() {
+        CORRADE_COMPARE(translations[objectId], (Vector3d{2.0f,0.0f,0.0f}));
+        CORRADE_COMPARE(visibilities[objectId], true);
+
+        Containers::Array<UnsignedInt> lights = scene->lightsFor(objectId);
+        CORRADE_COMPARE(lights.size(), 1);
+
+        Containers::Optional<LightData> light = importer->light(lights[0]);
+        CORRADE_VERIFY(light);
+        CORRADE_COMPARE(light->type(), LightData::Type::Point);
+        CORRADE_COMPARE(light->color(), (Color3{0.31f, 0.32f, 0.33f}));
+        CORRADE_COMPARE(light->intensity(), 3.0f);
+        CORRADE_COMPARE(light->range(), Constants::inf());
+        CORRADE_COMPARE(light->attenuation(), (Vector3{0.0f,1.0f,0.0f}));
+    }
+
+    {
+        Long objectId = importer->objectForName("spotLight1");
+        CORRADE_COMPARE(translations[objectId], (Vector3d{3.0f,0.0f,0.0f}));
+        CORRADE_COMPARE(importer->objectName(objectId), "spotLight1");
+        CORRADE_COMPARE(visibilities[objectId], true);
+
+        Containers::Array<UnsignedInt> lights = scene->lightsFor(objectId);
+        CORRADE_COMPARE(lights.size(), 1);
+
+        Containers::Optional<LightData> light = importer->light(lights[0]);
+        CORRADE_VERIFY(light);
+        CORRADE_COMPARE(light->type(), LightData::Type::Spot);
+        CORRADE_COMPARE(light->color(), (Color3{0.41f, 0.42f, 0.43f}));
+        CORRADE_COMPARE(light->intensity(), 4.0f);
+        CORRADE_COMPARE(light->range(), Constants::inf());
+        CORRADE_COMPARE(light->attenuation(), (Vector3{0.0f,0.0f,1.0f}));
+        CORRADE_COMPARE(light->innerConeAngle(), 12.0_degf);
+        CORRADE_COMPARE(light->outerConeAngle(), 80.0_degf);
+    }
+
+    {
+        Long objectId = importer->objectForName("areaLight1");
+        CORRADE_COMPARE(translations[objectId], (Vector3d{4.0f,0.0f,0.0f}));
+        CORRADE_COMPARE(importer->objectName(objectId), "areaLight1");
+        CORRADE_COMPARE(visibilities[objectId], true);
+
+        Containers::Array<UnsignedInt> lights = scene->lightsFor(objectId);
+        CORRADE_COMPARE(lights.size(), 1);
+
+        /* Area lights are not supported */
+        Containers::Optional<LightData> light = importer->light(lights[0]);
+        CORRADE_VERIFY(!light);
+    }
+
+    {
+        Long objectId = importer->objectForName("volumeLight1");
+        CORRADE_COMPARE(translations[objectId], (Vector3d{5.0f,0.0f,0.0f}));
+        CORRADE_COMPARE(importer->objectName(objectId), "volumeLight1");
+        CORRADE_COMPARE(visibilities[objectId], true);
+
+        Containers::Array<UnsignedInt> lights = scene->lightsFor(objectId);
+        CORRADE_COMPARE(lights.size(), 1);
+
+        /* Volume lights are not supported */
+        Containers::Optional<LightData> light = importer->light(lights[0]);
+        CORRADE_VERIFY(!light);
+    }
+
+    CORRADE_COMPARE(out.str(),
+        "Trade::UfbxImporter::light(): light type 3 is not supported\n"
+        "Trade::UfbxImporter::light(): light type 4 is not supported\n"_s);
```

Comment:
Oh, if you 

```cpp
#include <Corrade/Utility/DebugStl.h>
```

you don't need to use the `_s` when comparing string literals with stringstreams. Apologies for the messy halfway-done removal of STL strings :sweat_smile:

Reply:
I tried that earlier but got hit with this error so I just went with the `_s` everywhere:

```
...\magnum-plugins\src\MagnumPlugins\UfbxImporter\Test\UfbxImporterTest.cpp(176): message : see reference to function template instantiation 'Corrade::Containers::StridedArrayView<1,const Magnum::Vector3> Magnum::Trade::MeshData::attribute<Magnum::Vector3,void>(const Magnum::Trade::MeshAttribute,const Magnum::UnsignedInt) const' being compiled
...\corrade\src\Corrade/TestSuite/Comparator.h(274,52): error C2593: 'operator <<' is ambiguous
...\corrade\src\Corrade/Utility/Debug.h(626,16): message : could be 'Corrade::Utility::Debug &Corrade::Utility::Debug::operator <<(const Corrade::Containers::String &)'
...\corrade\src\Corrade/Utility/Debug.h(607,16): message : or       'Corrade::Utility::Debug &Corrade::Utility::Debug::operator <<(Corrade::Containers::StringView)'
```

Reply:
`Utility/DebugStl.h` (not `Containers/StringStl.h`) should solve that, hopefully. If not, don't bother, I'll check that after. Apologies for the mess, again :D

Reply:
Ah sorry misread, that did the trick! I noticed I'm a bit inconsistent with these myself, but which style do you prefer for passing strings into functions expecting `StringView`?

```cpp
Long objectId = importer->objectForName("ambientLight1");
Long objectId = importer->objectForName("ambientLight1"_s);
```

Reply:
While it somewhat matters in the plugin implementation (as the `_s` avoids a runtime `strlen()` call), it doesn't matter at all in the test. Go with what's shorter to type there.